lawnchair/src/app/lawnchair/ui/preferences/components/PreferenceSearchScaffold.kt

```diff
@@ -74,7 +74,7 @@ fun PreferenceSearchScaffold(
             }
         },
         bottomBar = { BottomSpacer() },
-        contentPadding = WindowInsets.systemBars.only(WindowInsetsSides.Horizontal).asPaddingValues()
+        contentWindowInsets = WindowInsets.systemBars.only(WindowInsetsSides.Horizontal)
```

Comment:
Not very sure about the migration of `Insets`, please let me know if there is anything missing in https://google.github.io/accompanist/insets/#migration

Reply:
Reverting this change in #3584.